app/templates/events/view.hbs

```diff
@@ -1,6 +1,59 @@
 {{#unless (includes session.currentRouteName 'events.view.edit')}}
-  {{!-- Tabs for the event view page will come here --}}
-  {{outlet}}
+  <div class="ui grid stackable">
+    <div class="row">
+      <div class="four wide column">
+        <h2 class="ui header">Fossasia Test</h2>
+      </div>
+      <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
+        {{#if device.isMobile}}
+          <button class="ui button"><i class="unhide icon"></i></button>
```

Comment:
Yes. This is exactly what is needed. In smaller screens the grouping doesn't make sense.